nx.json

```diff
@@ -112,7 +112,7 @@
     }
   },
   "namedInputs": {
-    "default": ["{projectRoot}/**/*", "sharedGlobals"],
+    "default": ["{projectRoot}/**/*", "sharedGlobals", "{workspaceRoot}/.github/workflows/pr.yml"],
```

Comment:
temporary cache invalidation that triggers affected on all proejcts if pr.yml has been touched